.github/workflows/build.yml

```diff
@@ -107,6 +111,23 @@ jobs:
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         docker push antrea/octant-antrea-ubuntu:latest
 
+  build-antrea-mc-controller-ubuntu:
+    needs: check-changes
+    if: ${{ needs.check-changes.outputs.has_changes == 'yes' || github.event_name == 'push' }}
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build antrea-mc-controller-ubuntu Docker image
+      run: make antrea-mc-controller-ubuntu
+    - name: Push antrea-mc-controller-ubuntu Docker image to registry
+      if: ${{ github.repository == 'antrea-io/antrea' && github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
I wonder if you want to use `'refs/heads/feature/multi-cluster'` here instead of `'refs/heads/main'`? Don't you want the image to be pushed to the registry when the feature branch is updated?

Reply:
yes, I want to push the image to registry, yesterday I tried to remote this condition but it failed, let me update ti feature branch to see if it can works or not. thanks.